service/gcsutils/gcstools/commoncli/common.go

```diff
@@ -79,6 +80,15 @@ func SetupLogging(args ...*string) error {
 	if len(args) < 1 {
 		return fmt.Errorf("Invalid log params")
 	}
-	utils.SetLoggingOptions("verbose", *args[0])
+
+	logrus.AddHook(commonutils.NewStackHook([]logrus.Level{logrus.InfoLevel, logrus.DebugLevel}))
```

Comment:
It seems that in the old behavior we would only log the stacks for a "verbose" level trace. But for commoncli this is not configurable from the cmdline so we "always" got traces. What this does is maintain that behavior such that any calls to ```Print{f}``` or ```Info{f}``` will get the traces but not any higher levels. 

If you want we can just do ```logrus.AllLevels``` for the hook here but we should probably make the log level configurable then and only enable that if we pass ```DebugLevel``` like I did in ```main.go```